### DISCUSSION TITLE
Github SSO

### DISCUSSION THREAD

**Discussion by xchen-ht** [community/community#34004] [Enterprise]
We are looking to implement Okta SSO with Github. What would happen for people outside of our organization? We cannot onboard them to our Okta instance. Can they stay as is to login with Github 2 factor authentication while rest of us use Okta SSO?

**Comment by djdefi** [comment]
Outside collaborators of GitHub Enterprise Cloud organizations are not required to authenticate through a SSO provider.

>SAML authentication is not required for outside collaborators.

ref: https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/preparing-to-enforce-saml-single-sign-on-in-your-organization

**Reply by lialex1202** [reply]
Will they still receive an email to click or a confirmation?
